src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -500,3 +503,106 @@ export async function refundRequestIsValid(
 
   return { valid: true };
 }
+
+export type SpokePoolEventFilter = {
+  originChainId?: number;
+  destinationChainId?: number;
+  relayer?: string;
+  fromBlock?: number;
```

Comment:
Why don't you also specify an optional toBlock? I'm missing some of the context on how this will be used

Reply:
We typically just default to `latest`, so I haven't added it. We can certainly add it later, but it didn't seem necessary for now.